en/docs/setup/migrating-data-publishers.md

```diff
@@ -3,47 +3,47 @@
 In prior versions of WSO2 Identity Server, data publishers were
 implementations of the
 [AbstractAuthenticationDataPublisher](https://github.com/wso2-extensions/identity-data-publisher-authentication/blob/master/components/org.wso2.carbon.identity.data.publisher.application.authentication/src/main/java/org/wso2/carbon/identity/data/publisher/application/authentication/AbstractAuthenticationDataPublisher.java)
-that are invoked iteratively by the
+which were invoked iteratively by the
 [AuthnDataPublisherProxy](https://github.com/wso2-extensions/identity-data-publisher-authentication/blob/master/components/org.wso2.carbon.identity.data.publisher.application.authentication/src/main/java/org/wso2/carbon/identity/data/publisher/application/authentication/AuthnDataPublisherProxy.java)
 when a session changes, such that the data publishers send events to
-their corresponding destinations. From WSO2 Identity Server 5.8.0
+their corresponding destinations. From WSO2 Identity Server 5.9.0
```

Comment:
we migrated the data publishers to event handlers in 5.8.0 right? I think that's why it mentions from 5.8.0 onwards. Shall we revert this?

Reply:
Fixed: 1325d3b997056bb2e11dd84500109c72375e078e